projects/Mallard/android/build.gradle

```diff
@@ -2,10 +2,10 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "28.0.3"
-        minSdkVersion = 16
-        compileSdkVersion = 28
-        targetSdkVersion = 28
+        buildToolsVersion = "29.0.2"
+        minSdkVersion = 17
+        compileSdkVersion = 29
+        targetSdkVersion = 29
```

Comment:
Is this all fine? Does this need to happen in this PR?

Reply:
yes, it's part of the masked element

Reply:
Cool thanks 👍